MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -10,154 +10,87 @@ final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
     }
     
-    private func showNextQuestionOrResults() {
-        if currentQuestionIndex == questionsAmount - 1 {
-            statisticService?.store(correct: correctAnswers, total: questionsAmount)
-            
-            let alertModel = AlertModel(
-                title: "Этот раунд окончен",
-                message: makeResultMessage(),
-                buttonText: "OK") { [weak self] _ in
-                    guard let self = self else { return }
-                    self.correctAnswers = 0
-                    self.currentQuestionIndex = 0
-                    self.questionFactory?.requestNextQuestion() }
-            alertPresenter?.showAlert(quiz: alertModel)
-            
-        } else { // если еще не конец игры
-            currentQuestionIndex += 1 // увеличиваем индекс вопроса
-            questionFactory?.requestNextQuestion()
-        }
+    func extinguishImageBorder() {
+        imageView.layer.borderWidth = 0
     }
     
-    private func showNetworkError(message: String) {
-        activityIndicator.stopAnimating()
+    func showNetworkError(message: String) {
+        hideLoadingIndicator()
         
-        let alertModel = AlertModel(
+        let networkError = AlertModel(
             title: "Ошибка",
             message: message,
             buttonText: "Попробовать еще раз") { [weak self] _ in
                 guard let self = self else { return }
-                self.correctAnswers = 0
-                self.currentQuestionIndex = 0
-                self.questionFactory?.loadData()
-                activityIndicator.startAnimating()
+                self.presenter.reloadGame()
+                self.showLoadingIndicator()
             }
-        alertPresenter?.showAlert(quiz: alertModel)
+        alertPresenter?.showAlert(quiz: networkError)
     }
     
-    private func makeResultMessage() -> String {
-        guard let statisticService = statisticService else { return "" }
-        
-        let average = "Средняя точность: \(String(format: "%.2f", statisticService.totalAccuracy))%"
-        let bestGameInfoLine = "Рекорд: \(statisticService.bestGame.correct)/\(statisticService.bestGame.total) (\(statisticService.bestGame.date.dateTimeString))"
-        let currentGameResult = "Ваш результат: \(correctAnswers)/\(questionsAmount)"
-        let totalGamesPlayed = "Количество сыгранных квизов: \(statisticService.gamesCount)"
-        
-        let resultMessage = [currentGameResult,
-                             totalGamesPlayed,
-                             bestGameInfoLine,
-                             average].joined(separator: "\n")
-        return resultMessage
+    func showResult(message: String) {
+        let alertModel = AlertModel(
+            title: "Этот раунд окончен",
+            message: message,
+            buttonText: "OK") { [weak self] _ in
+                guard let self = self else { return }
+                self.presenter.restartGame() }
```

Comment:
Фигурные скобки закрываются на том же уровне, что и открывающая их строчка (button Text). Либо остается на той же строке что и открывающая. 
В твоем случае так будет правильнее:
```suggestion
            buttonText: "OK") { [weak self] _ in
                guard let self = self else { return }
                self.presenter.restartGame() 
            }
```